src/help/util.ts

```diff
@@ -43,7 +43,7 @@ function collateSpacedCmdIDFromArgs(argv: string[], config: IConfig): string[] {
   const findId = (argv: string[]): string | undefined => {
     const final: string[] = []
     const idPresent = (id: string) => ids.includes(id)
-    const isFlag = (s: string) => s.startsWith('-')
+    const isFlag = (s: string) => s.startsWith('-') || s.includes('=')
```

Comment:
@mdonnalley not sure this is the right place to do this. It catches var args with the `key=value` pattern, but not simple args.